bot/client.py

```diff
@@ -127,25 +127,40 @@ async def cog_command_error(self, context, error):
         await context.send('An error occurred: {}'.format(str(error)))
         logger.error('An error occurred: {}'.format(str(error)))
 
-    @commands.command(invoke_without_subcommand=True)
+    @commands.hybrid_command(
+        name='join',
+        invoke_without_subcommand=True,
+        with_app_command=True,
+        description='Make Myinstants bot join.',
+    )
     async def join(self, context):
         if not context.author.voice:
             raise VoiceError('You are not connected to a voice channel.')
 
+        await context.send('Joining user\'s channel.')
         channel = context.author.voice.channel
         if context.voice_state.voice:
             return await context.voice_client.move_to(channel)
         context.voice_state.voice = await channel.connect()
 
-    @commands.command(aliases=['disconnect', 'stop'])
+    @commands.hybrid_command(
+        name='leave',
+        with_app_command=True,
+        description='Disconnect Myinstants bot.'
+    )
     async def leave(self, context):
         if not context.voice_state.voice:
             return await context.send('Not connected to any voice channel.')
 
+        await context.send('Leaving current channel.')
```

Comment:
Same case here. This early feedback can break all the following code in the function

Reply:
This is not breaking the code and the feedback is important!